server/planning/events/events_template.py

```diff
@@ -220,15 +223,15 @@ def get(self, req, lookup):
         templates = list(
             app.data.mongo.pymongo(resource="events_template")
             .db["events_template"]
-            .find({"_id": {"$in": templates_ids}})
+            .find({"_id": {"$in": templates_ids}})  # TODO-ASYNC : Confirm if to be converted to find_async here
```

Comment:
You can use `app.data.mongo_async` here instead (example from [desks resource](https://github.com/superdesk/superdesk-core/blob/async/apps/desks.py#L722)). It returns an instance of `superdesk.eve_async.service.AsyncBaseService`